postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -469,3 +474,19 @@ def _explain_pg_stat_activity(self, rows):
                     1,
                     tags=self._tags + ["error:collect-plan-for-statement-crash"],
                 )
+
+    def _get_query_max_text_size(self):
+        with self._get_db(self._config.dbname).cursor(cursor_factory=psycopg2.extras.DictCursor) as cursor:
+            self._log.debug("Running query [%s]", MAX_ACTIVITY_QUERY_SIZE_QUERY)
+            cursor.execute(MAX_ACTIVITY_QUERY_SIZE_QUERY)
+            row = cursor.fetchone()
+            return int(row['setting'])
```

Comment:
Can this value ever be null/None? Which version was this setting added?

Reply:
Looks like this was [introduced in 8.4](https://www.postgresql.org/docs/8.4/release-8-4.html). If we need to support versions before that, I need to figure out what the right default value would be. I would imagine it was just defaulting to 1024 before the introduction of the config but I'd have to dig and make sure.

Reply:
I forgot to look deeper into this with all the issues with the multi-byte characters I was working on yesterday but are we good if all the versions tested in the integration are passing? If that's the extent of our supported versions, I think the fact that tests are all working fine gives us the answer we need?